opm/simulators/timestepping/TimeStepControl.cpp

```diff
@@ -73,7 +74,7 @@ namespace Opm
     }
 
     double SimpleIterationCountTimeStepControl::
-    computeTimeStepSize( const double dt, const int iterations, const RelativeChangeInterface& /* relativeChange */, const double /*simulationTimeElapsed */) const
+    computeTimeStepSize( const double dt, const int iterations, const RelativeChangeInterface& /* relativeChange */, const AdaptiveSimulatorTimer& substepTimer) const
```

Comment:
Note: The `subStepTimer` is unused in this function (`SimpleIterationCountTimeStepControl::computeTimeStepSize()`) so I suggest you either remove the name or tag the parameter as `[[maybe_unused]]`.